tests/roompanel/TestRoomWidget.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (C) 2023 The MMapper Authors
+
+#include "roompanel/RoomWidget.h"
+#include <QObject>
+
+class TestRoomWidget final : public QObject
+{
+    Q_OBJECT
+public:
+    TestRoomWidget();
+    ~TestRoomWidget() final;
+
+private:
+private Q_SLOTS:
+    void testRoomModelInitialization();
```

Comment:
If easy, could you add another test here where you add an actual mob to the Widget and verify it worked end to end?

Reply:
I love the idea, ideally it would be a test where GMCP message is input and visual feedback is the assertion. Unfortunately I could not figure out how to easily do it. 

I had to delete TestRoomWidget because it had a dependency that produced a cryptic MinGW compilation error. It only contained 1 test that was not very good, so not a big loss. Most of the testable logic is in TestRoomManager anyway.

There are a little more comprehensive tests in TestRoomManager.